hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/CreatePrecompileSuite.java

```diff
@@ -652,20 +602,18 @@ final Stream<DynamicTest> createTokenWithDefaultExpiryAndEmptyKeys() {
                 newKeyNamed(THRESHOLD_KEY)
                         .shape(THRESHOLD_KEY_SHAPE.signedWith(sigs(ED25519_ON, TOKEN_CREATE_CONTRACT))),
                 cryptoCreate(ACCOUNT).balance(ONE_MILLION_HBARS).key(THRESHOLD_KEY),
-                withOpContext((spec, opLog) -> allRunFor(
-                        spec,
-                        contractCall(TOKEN_CREATE_CONTRACT, tokenCreateContractAsKeyDelegate)
-                                .via(FIRST_CREATE_TXN)
-                                .gas(GAS_TO_OFFER)
-                                .sending(DEFAULT_AMOUNT_TO_SEND)
-                                .payingWith(ACCOUNT)
-                                .exposingResultTo(result -> {
-                                    log.info(EXPLICIT_CREATE_RESULT, result[0]);
-                                    final var res = (Address) result[0];
-                                    createdTokenNum.set(numberOfLongZero(HexFormat.of()
-                                            .parseHex(res.toString().substring(2))));
-                                })
-                                .hasKnownStatus(SUCCESS))),
+                contractCall(TOKEN_CREATE_CONTRACT, tokenCreateContractAsKeyDelegate)
+                        .via(FIRST_CREATE_TXN)
+                        .gas(GAS_TO_OFFER)
+                        .sending(DEFAULT_AMOUNT_TO_SEND)
+                        .payingWith(ACCOUNT)
+                        .exposingResultTo(result -> {
+                            log.info(EXPLICIT_CREATE_RESULT, result[0]);
+                            final var res = (Address) result[0];
+                            createdTokenNum.set(numberOfLongZero(
+                                    HexFormat.of().parseHex(res.toString().substring(2))));
```

Comment:
I recognize you're only moving what's already there, but since this code is changing anyway:
```suggestion
                                    // Remove the leading '0x'
                                    HexFormat.of().parseHex(res.toString().substring(2))));
```